cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -254,115 +251,6 @@ public void testProgramStartStopStatus() throws Exception {
     Assert.assertEquals(200, response.getResponseCode());
   }
 
-  @Test
```

Comment:
Needed to remove these tests - since we don't have non versioned apps (default versioned apps) or explicitly version defined apps.